backend/api/src/main/kotlin/software/shonk/adapters/incoming/ShorkInterpreterController.kt

```diff
@@ -78,9 +78,12 @@ fun Route.configureShorkInterpreterControllerV1() {
     }
 
     post("/lobby") {
-        val lobbyId = shorkUseCase.createLobby()
-        call.respond(HttpStatusCode.Created, lobbyId.toString())
-        return@post
+        val creatingPlayerName = call.receive<String>()
```

Comment:
I think this won't work with the JSON we receive:

We send `
{
    "playerName": String,
}
`

and that is what gets read by receive<String>

![2024-11-09-154422_hyprshot](https://github.com/user-attachments/assets/003cf253-bca5-42a9-8f8b-bac58d773335)

You would have to parse out the needed field by hand I think.

Reply:
This hasn't been resolved I think